### PR TITLE
Fix stack for windows

### PR DESCRIPTION
**Pull request by kderme**
Related https://github.com/input-output-hk/ouroboros-network/issues/1082
Continuation of https://github.com/input-output-hk/ouroboros-network/pull/1774, which makes use of https://github.com/input-output-hk/cardano-prelude/pull/100

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -91,6 +94,7 @@ extra-deps:
 
   # Windows only
   - Win32-2.6.2.0
+  - hedgehog-1.0.1@sha256:b65b1ebde7b7fe8e693070916e45aa2ed63eea5e3f84ac3a8f9a0b7f8fcf0337
```
Comment (mrBliss) [inline stack.yaml]: Why is this needed? The `snapshot.yaml` in `cardano-prelude` mentions `hedgehog-1.0.2`, so this is overriding it. Also, the `cabal.project` does not mention this constraint.

**Comment by kderme** [comment]
@mrBliss with this, `ouroboros-consensus` still depends on `3893f21db1665e7bab9e8c6504c547fc7c28f98c` of prelude, but uses the resolver of `fe76ec64f6b45259cc407a6d840dad79ee6063b6`. Do you think this is an inconsistency we should fix? I just though bumping also the dependency could mean more cascading changes.

File: stack.yaml (hunk 1 above)
Comment (kderme) [reply stack.yaml]: My bad. I probably first tried stack on a pretty old commit which mentioned `hedgehog-1.0`

**Comment by mrBliss** [comment]
@kderme Yes, it's an inconsistency that should be fixed.

File: stack.yaml (hunk 1 above)
Comment (mrBliss) [reply stack.yaml]: Related: #1796

**Comment by kderme** [comment]
stack and cabal work on windows

**Comment by mrBliss** [comment]
FYI, the test failure on Hydra is transient and the sign of an overloaded CI machine (the test involves time).
